.symfony.bundle.yaml

```diff
@@ -1,3 +1,3 @@
-branches: ["1.x", "1.23.x"]
-maintained_branches: ["1.x", "1.23.x"]
+branches: ["2.x", "1.x"]
+maintained_branches: ["2.x", "1.x"]
```

Comment:
I put the `2.x` in first position, I don't know how it would impact the order in the symfony docs. I'm assuming the first one is the default one